core/examples/luigi-sample-angular/src/assets/basicConfiguration.js

```diff
@@ -28,7 +28,7 @@ Luigi.setConfig({
     /**
      * Development:
      * For hash routing, set to true and run `npm run start`
```

Comment:
This made me think I had to run npm start again if I change the routing strategy, which is not the case after we merge this task. I would recommend to remove the _and run `npm run start`_ part

Reply:
good idea. done